Week1/Meetup/index.js

```
@@ -0,0 +1,154 @@
+});
+
+app.get("/createmeetup", (req, res) => {
+  const sql = "CREATE DATABASE meetup";
+
+  db.query(sql, (err, result) => {
+    if (err) throw err;
+    console.log(result);
+    db.changeUser({ database: "meetup" });
+    res.send("Meetup created");
+  });
+});
+
+app.get("/deletemeetup", (req, res) => {
+  const sql = "DROP DATABASE meetup";
+
+  db.query(sql, (err, result) => {
+    if (err) throw err;
+    console.log(result);
+    res.send("Database deleted");
+  });
+});
+
+app.get("/createinvitee", (req, res) => {
+  const sql = `CREATE TABLE IF NOT EXISTS Invitee(
+        id INT AUTO_INCREMENT,
+         invitee_no INT, 
+         invitee_name VARCHAR(100),
+         invited_by VARCHAR(100),
+         PRIMARY KEY(id))`;
+
+  db.query(sql, (err, result) => {
+    if (err) throw err;
+    console.log(result);
+    res.send("Invitee created");
+  });
+});
+
+app.get("/createroom", (req, res) => {
+  const sql = `CREATE TABLE IF NOT EXISTS Room(
+        id INT AUTO_INCREMENT,
+        room_no INT, 
+        room_name VARCHAR(100),
+        floor_number INT,
+         PRIMARY KEY(id))`;
+
+  db.query(sql, (err, result) => {
+    if (err) throw err;
+    console.log(result);
+    res.send("ROOM created");
+  });
+});
+
+app.get("/createmeeting", (req, res) => {
+  const sql = `CREATE TABLE IF NOT EXISTS Meeting(
+        id INT AUTO_INCREMENT,
+        meeting_no INT, 
+        meeting_title VARCHAR(100),
+        starting_time TIMESTAMP,
+        ending_time TIMESTAMP,
+        room_no INT,
+         PRIMARY KEY(id))`;
+
+  db.query(sql, (err, result) => {
+    if (err) throw err;
+    console.log(result);
+    res.send("Meeting created");
+  });
+});
+
+app.get("/addroom", (req, res) => {
+  const values = [
+    [10, "East room", 1],
+    [21, "West room", 2],
+    [32, "South room", 3],
+    [43, "North room", 4],
+    [54, "Middle room", 5],
+  ];
+  const sql = `INSERT IGNORE INTO room (room_no, room_name, floor_number) values ?`;
```

INSERT IGNORE is a bit dangerous because you are basically ignoring errors instead of making sure they don't happen or dealing properly with them if they do happen.